src/System.Private.CoreLib/src/System/Threading/RuntimeThread.cs

```diff
@@ -2,33 +2,123 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
+using System.Diagnostics;
+using System.Runtime;
+using Microsoft.Win32.SafeHandles;
 
 namespace System.Threading
```

Comment:
I am simply asking to not remove the old file so I can use it as a wrapper.  If the file is deleted, I'll have to recreate it until we get an updated package from CoreFX.  The actual implementation should go into the right place, I.R.A.RuntimeThread.